Casks/j/jslegendre-themeengine.rb

```diff
@@ -9,10 +9,13 @@
 
   livecheck do
     url :url
-    strategy :git do |tags|
+    regex(/^v?(\d+(?:\.\d+)*)\((\d+)\)$/i)
+    strategy :git do |tags, regex|
```

Comment:
```suggestion
    strategy :github_latest do |json, regex|
```

Upstream has marked all releases before the latest (v1.0.0(118)) as pre-release, so we should probably use `GithubLatest` here.